hack/e2e.sh

```diff
@@ -22,6 +22,25 @@ fi
 echo "IMAGE=${IMAGE}"
 echo "IMAGE_TAG=${IMAGE_TAG}"
 
+function backoff() {
+    local max_attempts=10
+    local attempt=0
+    local failed=0
+    for i in $(seq 0 ${max_attempts}); do
+        "$@" && failed=0 || failed=1
+        if [[ $failed -eq 0 ]]; then
+            break
+        fi
+        attempt=$(( attempt + 1 ))
+        if [[ $attempt -gt 5 ]]; then
```

Comment:
do we need `${max_attempts}` as we are breaking at 5?

Reply:
Ah, good catch. Replaced `for` loop with `while true` and 5 with `max_attempts`